src/main/resources/db/migration/V2023.10.25.08.58.36__update_doc_type_label_default.sql

```diff
@@ -0,0 +1 @@
+UPDATE user_files SET doc_type_label='${user_file_doc_type_default_label}';
```

Comment:
There's a `11.28` transmission-related migration file thats in main rn - Can we rename this file to a date after that to avoid out-of-order migration errors/warnings?